### PR TITLE
fix recomputeStats

### PR DESCRIPTION
**Pull request by kuboon**
`section.hit` and `section.found` are always null.
it overwrides hit&found by 0 if details are empty.

### CODE REVIEW

**Comment by tenninebt** [comment]
Will review today. Sorry for the delay.

**Comment by tenninebt** [comment]
@kuboon have an example coverage file? as I'm thinking the fix might be improved to cover more cases.

File: src/coverage-parser.ts [hunk 1]
```diff
@@ -2,12 +2,12 @@ import { parseContent as parseContentClover } from "@cvrg-report/clover-json";
 import * as vscodeLogging from '@vscode-logging/logger';
 import { parseContent as parseContentCobertura } from "@cvrg-report/cobertura-json";
 import { parseContent as parseContentJacoco } from "@cvrg-report/jacoco-json";
-import { Section, source } from "lcov-parse";
+import { LcovFile, source } from "lcov-parse";
```
Comment (kuboon) [inline src/coverage-parser.ts:5]: I found `LcovFile` here. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/lcov-parse/index.d.ts
I could not find any keyword `Section` from lcov-parse module.

**Comment by kuboon** [comment]
Added type checking for clearing what was the problem.

**Comment by tenninebt** [comment]
Merged the same change in #32

**Pull request closed without merging**